src/main/webapp/resources/js/pages/projects/associated-projects/ViewAssociatedProjects.jsx

```diff
@@ -0,0 +1,114 @@
+import React, { useEffect, useState } from "react";
```

Comment:
File comment

Reply:
Updated in 6262caf9c9e3bddc898cb212dca62e4f0b1fa543

Reply:
Updated in c7edce73cbc8f57b149a0692c5b2d738fcbda5d5